website/content/docs/concepts/adaptive-overload-protection/index.mdx

```diff
@@ -60,21 +58,24 @@ In some cases, a sudden influx of write requests that exceeds Vault's hardware
 capacity can result in the writes queueing for so long that every request times
 out before the write can make it through the queue. This makes Vault effectively
 unavailable to clients even though it is still processing requests and storing
-data as fast as it can. This is illustrated in the test results shown below for
-a workload of 100% logins.
+data as fast as it can. This can also result in Replication lagging behind causing
```

Comment:
```suggestion
data as fast as it can. This can also result in replication lagging behind causing
```
nit: but we don't treat "replication" as a proper noun.